projects/plugins/crm/.eslintrc.js

```diff
@@ -0,0 +1,28 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const loadIgnorePatterns = require( 'jetpack-js-tools/load-eslint-ignore.js' );
+
+module.exports = {
+	root: true,
+	extends: [
+		require.resolve( 'jetpack-js-tools/eslintrc/base' ),
+		require.resolve( 'jetpack-js-tools/eslintrc/wp-eslint-plugin/recommended' ),
+	],
+	ignorePatterns: loadIgnorePatterns( __dirname ),
+	rules: {
+		camelcase: 0,
+		'no-var': 0,
+		'prefer-const': 0,
+		'prettier/prettier': 0,
+		// This is not a react project.
+		'react-hooks/rules-of-hooks': 0,
+
+		// Don't require JSDoc on functions initially.
+		'jsdoc/check-tag-names': 0,
+		'jsdoc/require-jsdoc': 0,
+		'jsdoc/require-returns': 0,
+		'jsdoc/require-description': 0,
+		'jsdoc/require-property-description': 0,
+		'jsdoc/require-param-description': 0,
+		'jsdoc/require-param-type': 0,
+	},
+};
```

Comment:
FYI, you should be able to simplify this by overriding the parent config instead of replacing it entirely.
```suggestion
module.exports = {
	extends: [
		require.resolve( 'jetpack-js-tools/eslintrc/wp-eslint-plugin/recommended' ),
	],
	rules: {
		camelcase: 0,
		'no-var': 0,
		'prefer-const': 0,
		'prettier/prettier': 0,
		// This is not a react project.
		'react-hooks/rules-of-hooks': 0,

		// Don't require JSDoc on functions initially.
		'jsdoc/check-tag-names': 0,
		'jsdoc/require-jsdoc': 0,
		'jsdoc/require-returns': 0,
		'jsdoc/require-description': 0,
		'jsdoc/require-property-description': 0,
		'jsdoc/require-param-description': 0,
		'jsdoc/require-param-type': 0,
	},
};
```
OTOH, it looks like it doesn't make a difference currently since you have entries in `.eslintignore` that are excluding almost all your JS files from linting.

Reply:
Thanks Brad, makes sense. I've now checked the very first commit when CRM was moved into the monorepo, and that is exactly when the variable declarations changed. So it was on that first commit, before the .eslintignore was in place, that the auto linting happened. 

So yeah I could just remove this - and add the webpack config file to the .eslintignore list as the `'jsdoc/check-tag-names'` helped with that.

Reply:
Ideally we should work on removing files from the eslintignore rather than adding more. 😀 It's even more of a way to hide problems than `tools/eslint-excludelist.json`, mainly it should be used to stop eslint from trying to lint built files or test fixtures.

But that's a much bigger project, of course.